libr/anal/sign.c

```diff
@@ -2277,6 +2332,184 @@ static int match_metrics(RSignItem *it, void *user) {
 	return 0;
 }
 
+static int _sig_to_vec_cb(RSignItem *it, void *user) {
+	if (r_pvector_push ((RPVector *)user, it)) {
+		return 1;
+	} else {
```

Comment:
no need to else after return, but you can also use the ternary operator and make oneliner here